### DISCUSSION TITLE
New variation of Hardrock - Extreme Hardrock

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#14102] [Ideas] [answered]
Increases parameters to 11, and also reflects the map in a random direction each time. 

This may not be a separate mod, but a second kind of hardcore.

**Comment by smoogipoo** [comment, marked as answer]
Use the mirror + difficulty adjust mod to achieve this, too much random pollution of HR mod.

**Reply by Moltanicaa** [reply]
Boring, but you right
